### PR TITLE
feat: add truncateAddress utility function

### LINKED ISSUE #90: Add address shortening utility function
## Problem

We often need to shorten Ethereum addresses for display purposes (e.g., in UIs) to formats like `0xcafe...beef` or `0xca...ef`. Currently, the SDK doesn't provide a utility function for this common use case.

> [!NOTE]
> Viem doesn't currently support this feature: https://github.com/wevm/viem/discussions/1626

## Solution

Add a `truncateAddress` utility function to the SDK that:

- **Default format**: `0xcafe...beef` (4 characters after `0x` prefix + last 4 characters)
- **Configurable**: Accept optional parameter to specify character count per side
- **Graceful handling**: Return original string for invalid inputs or addresses too short to shorten
- **Type-safe**: Use `Sablier.EVM.Address` type (`0x${string}`)

**Example usage**:
\`\`\`typescript
import { truncateAddress } from "sablier";

truncateAddress("0x1234567890abcdef1234567890abcdef12345678"); 
// Returns: "0x1234...5678"

truncateAddress("0x1234567890abcdef1234567890abcdef12345678", 6); 
// Returns: "0x123456...345678"

truncateAddress("0x123"); 
// Returns: "0x123" (too short, returns original)
\`\`\`

**Implementation details**:
- Add function to `src/helpers.ts` (existing utility file)
- Include JSDoc comments with examples
- Follow existing code style and patterns
- Export automatically via main entry point

> [!TIP]
> Consider adding tests for edge cases: empty strings, non-hex characters, addresses without 0x prefix, and very short addresses.

## Files Affected

<details><summary>Toggle to see affected files</summary>
<p>

- [src/helpers.ts](https://github.com/sablier-labs/sdk/blob/main/src/helpers.ts)
- [tests/helpers.test.ts](https://github.com/sablier-labs/sdk/blob/main/tests/helpers.test.ts)

</p>
</details>

### PR DESCRIPTION
**Pull request by PaulRBerg**
## Summary

Added `truncateAddress()` helper to format Ethereum and Solana addresses for display purposes (e.g., `0xcafe...beef`, `DYw8...NSKK`). Supports configurable character count with sensible defaults (4 chars per side).

> [!NOTE]
> This utility handles both EVM (0x-prefixed) and Solana (base58) addresses automatically by detecting the prefix.

## Changes

- New `truncateAddress()` function in `src/helpers.ts`
- Comprehensive test coverage (12 new test cases)
- Gracefully handles edge cases (invalid/short addresses)

Closes #90

### CODE REVIEW

File: src/helpers.ts [hunk 1]
```diff
@@ -69,3 +69,53 @@ export function getNestedValues<T extends Record<string, unknown>>(obj: T): stri
...
+    return address;
+  }
+
+  // Handle Ethereum addresses (0x-prefixed)
+  if (address.startsWith("0x")) {
+    // Calculate minimum length needed: "0x" + chars on each side
+    const minLength = 2 + chars * 2;
+
+    // Return original if too short to truncate meaningfully
+    if (address.length <= minLength) {
```
[the remark this answers is not in the document]
Comment (PaulRBerg) [reply src/helpers.ts]: this is fine
